### PR TITLE
add change status test

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

**Comment by itkg-rboessel** [comment]
:+1:

**Comment by itkg-nanne** [comment]
:+1:

**Comment by nicolasThal** [comment]
@itkg-rboessel @itkg-nanne pr updated

**Comment by itkg-rboessel** [comment]
:+1:

**Comment by nicolasThal** [comment]
@itkg-nanne I have updated the PR to modify the error display on the route pattern validation

File: BackofficeBundle/Resources/views/BackOffice/Underscore/flashBag._tpl.twig [hunk 1]
```diff
@@ -1,3 +1,13 @@
 <div class="flashbag">
-    <%= html %>
+    <% if (typeof errors.error != 'undefined') { %>
```
Comment (itkg-nanne) [inline BackofficeBundle/Resources/views/BackOffice/Underscore/flashBag._tpl.twig]: this code is duplicate from Braincrafted and will not be synchronized on update. The reason why i did the render with Braincrafted.

File: BackofficeBundle/Validator/Constraints/BlockNodePatternValidator.php [hunk 2]
```diff
@@ -80,7 +63,7 @@ protected function getRefBlock(AreaContainerInterface $container)
     {
         $blockRef = array();
         $areas = $container->getAreas();
-        if (count($areas) > 0){
+        if ($container instanceof NodeInterface || count($areas) > 0){
```
Comment (itkg-nanne) [inline BackofficeBundle/Validator/Constraints/BlockNodePatternValidator.php]: Why adding this as areas are mandatory for NodeInterface?

Comment (nicolasThal) [inline BackofficeBundle/Validator/Constraints/BlockNodePatternValidator.php]: I added this test to avoid the case where you have a node with no areas

Comment (itkg-nanne) [inline BackofficeBundle/Validator/Constraints/BlockNodePatternValidator.php]: This case should not exist.

File: BackofficeBundle/Resources/views/BackOffice/Underscore/flashBag._tpl.twig (hunk 1 above)
Comment (itkg-nanne) [inline BackofficeBundle/Resources/views/BackOffice/Underscore/flashBag._tpl.twig]: You can't change this tpl without changing widget/statusView.coffee which use it too.

Comment (itkg-nanne) [inline BackofficeBundle/Resources/views/BackOffice/Underscore/flashBag._tpl.twig]: This code is used in submitAdmin.coffee too. You have to change both or none.

Comment (nicolasThal) [inline BackofficeBundle/Resources/views/BackOffice/Underscore/flashBag._tpl.twig]: I haven't saw it... thanks!!

**Comment by nicolasThal** [comment]
@itkg-nanne I have created a view to display the api errors, this way, if you are having an html response, you just display the response, on the other case we look througt the json response

**Comment by itkg-nanne** [comment]
:+1

**Pull request merged**